app/models/skin.rb

```diff
@@ -13,7 +13,7 @@ class Skin < ApplicationRecord
                  ]
 
   # any media types that are not a single alphanumeric word have to be specially handled in get_media_for_filename/parse_media_from_filename
-  MEDIA = %w(all screen handheld speech print braille embossed projection tty tv) + ['only screen and (max-width: 42em)'] + ['only screen and (max-width: 62em)']
+  MEDIA = %w(all screen handheld speech print braille embossed projection tty tv) + ['only screen and (max-width: 42em)', 'only screen and (max-width: 62em)', '(prefers-color-scheme: dark)', '(prefers-color-scheme: light)']
```

Comment:
Can you break this into multiple lines?

Reply:
How do we want that indented?

```
  MEDIA = %w(all screen handheld speech print braille embossed projection tty tv) +
    ['only screen and (max-width: 42em)', 'only screen and (max-width: 62em)',
    '(prefers-color-scheme: dark)', '(prefers-color-scheme: light)']
```

or something else?

Reply:
```ruby
  MEDIA = %w[all screen handheld speech print braille embossed projection tty tv] + [
    'only screen and (max-width: 42em)',
    'only screen and (max-width: 62em)',
    '(prefers-color-scheme: dark)',
    '(prefers-color-scheme: light)'
  ]
```
I checked, Hound wouldn't complain about this one.